kolibri/content/models.py

```diff
@@ -64,7 +64,24 @@ class ContentNode(MPTTModel):
     tags = models.ManyToManyField(ContentTag, symmetrical=False, related_name='tagged_content', blank=True)
 
     title = models.CharField(max_length=200)
+
+    # the instance_id is used for mapping a node between kolibri and the
+    # content curation server. We can't use the raw id, since ids aren't
+    # guaranteed to be consistent across different content curation servers
+    # (once we have a distributed content curation server), and content ids may
+    # be the same across different nodes within the same channel (for student
+    # logging and analytics purposes.) Apart from generating the instance_id
+    # upon node creation, we also change the instance_id whenever we move or
+    # copy a node).
+    instance_id = UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
On the bright side, this might also make it easier to do content DB exports within a single transaction, as you already know all the target PKs.

If we're being forward-looking to `kolibri-studio` (and distributed channel editing), then every model (at least those that are editable) should really have UUID pk's...